moped-editor/src/views/projects/projectView/ProjectMilestones.js

```diff
@@ -291,9 +290,7 @@ const ProjectMilestones = ({ projectId, loading, data, refetch }) => {
         </Typography>
       }
       options={{
-        ...(data.moped_proj_milestones.length < PAGING_DEFAULT_COUNT + 1 && {
-          paging: false,
-        }),
+        paging: false,
```

Comment:
this also fixed the bug where if you added more than one template worth of milestones, you'd get a white screen

Reply:
✨✨ ✨ ✨ ✨